client/lib/abtest/index.js

```diff
@@ -71,6 +71,8 @@ const parseDateStamp = datestamp => {
 const languageSlugs = map( config( 'languages' ), 'langSlug' );
 const langSlugIsValid = slug => languageSlugs.indexOf( slug ) !== -1;
 
+const crypto = require( 'crypto' );
```

Comment:
Please don't use the `crypto` library -- it's a Node polyfill that's big and evil. As @jonathansadowski points out, there is a better option: `hash.js/lib/hash/sha/1` that's much much smaller. You'll need to use SHA1 instead of MD5, but that shouldn't matter at all in this case.

Using a simple prime-and-modulus hash algorithm that @nylen suggested is also a nice option. This feature doesn't need a hash that has any cryptographic properties.